setup.py

```diff
@@ -37,7 +29,7 @@
     ],
     install_requires=[
         # We cannot use the git+https syntax here because of docker build issues
```

Comment:
```suggestion
        # We cannot use the git+https / git+ssh syntax here because of docker build issues
        # related to git not being installed
```